pallets/alt-vr/src/tests/extrinsics.rs

```diff
@@ -0,0 +1,130 @@
+use super::mock::*;
+use crate::tests::mock;
+use crate::{AltvrData, Error};
+use frame_support::{assert_noop, assert_ok};
+use frame_system::RawOrigin;
+
+#[test]
+fn create_altvr_happy() {
+    ExtBuilder::default().build().execute_with(|| {
+        let alice: mock::Origin = RawOrigin::Signed(ALICE).into();
+
+        assert_eq!(Altvr::altvrs(1), None);
+
+        let user_name: Vec<u8> = "sub".into();
+        let vchat_name: Vec<u8> = "strate".into();
+        let altvr_data = AltvrData::new(user_name.clone(), vchat_name.clone());
+        assert_ok!(Altvr::create_altvr(
+            alice.clone(),
+            user_name.clone(),
+            vchat_name.clone()
+        ));
+        assert_eq!(Altvr::altvrs(&ALICE), Some(altvr_data));
+    });
+}
+
+#[test]
+fn create_altvr_unhappy() {
```

Comment:
Don't forget to update the name of test.

Reply:
done